infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -757,373 +946,60 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
-
-        // Calculate the amount to mint to the reward address
-        rewardMintAmount = (reward * bulkSubmission.winningKeyCount);     
-
-        // mark the submission as claimed
-        bulkSubmission.claimed = true;
-        // increment the amount claimed on the challenge
-        challenges[_challengeId].amountClaimedByClaimers += rewardMintAmount;    
-    
-        esXai(esXaiAddress).mint(_bulkAddress, rewardMintAmount);
-
-        // Increment the total claims of this address
-        _lifetimeClaims[_bulkAddress] += rewardMintAmount;
-
-        // unallocate the tokens that have now been converted to esXai
-        _allocatedTokens -= rewardMintAmount;
-        emit BulkRewardsClaimed(_challengeId, _bulkAddress, rewardMintAmount, bulkSubmission.winningKeyCount);
-    }
-
-    /**
-    * @notice Get the maximum available stake amount, the lower of the actual stake amount and the maximum capacity amount
-    * @dev this will check the pool or the owners V1 stake
-    * @param assignedPool The address of the assigned pool, can be address(0) for keys that are not assigned.
-    * @param licenseOwner The key owner for V1 stake, can be address(0) if the key is assigned to a pool.
-    */
-    function getMaxStakedAmount(address assignedPool, address licenseOwner) internal view returns (uint256 stakedAmount) {      
-		stakedAmount = stakedAmounts[assignedPool];
-		if (assignedPool == address(0)) {
-			stakedAmount = stakedAmounts[licenseOwner];
-			uint256 ownerUnstakedAmount = NodeLicense(nodeLicenseAddress).balanceOf(licenseOwner) - assignedKeysOfUserCount[licenseOwner];
-			if (ownerUnstakedAmount * maxStakeAmountPerLicense < stakedAmount) {
-				stakedAmount = ownerUnstakedAmount * maxStakeAmountPerLicense;
-			}
-		} else { 
-			if (assignedKeysToPoolCount[assignedPool] * maxStakeAmountPerLicense < stakedAmount) {
-				stakedAmount = assignedKeysToPoolCount[assignedPool] * maxStakeAmountPerLicense;
-			}
-		}
     }
 
     //TEST FUNCTION this is used only for test coverage
-    // function toggleAssertionChecking() public {
-    //     isCheckingAssertions = !isCheckingAssertions;
-    //     emit AssertionCheckingToggled(isCheckingAssertions);
-    // }
+    // Leaving these functions in the contract for test coverage purposes
+    // In the future these can be removed if size becomes an issue
+    function toggleAssertionChecking() public {
+        isCheckingAssertions = !isCheckingAssertions;
+        emit AssertionCheckingToggled(isCheckingAssertions);
+    }
 
     //TEST FUNCTION this is used only for test coverage
-    // function setRollupAddress(address newRollupAddress) public {
-    //     rollupAddress = newRollupAddress;
-    //     emit RollupAddressChanged(newRollupAddress);
-    // }
-}
\ No newline at end of file
+    function setRollupAddress(address newRollupAddress) public {
+        rollupAddress = newRollupAddress;
+        emit RollupAddressChanged(newRollupAddress);
+    }
+}
```

Comment:
These need to be commented out in version 9 too !

Reply:
Should we comment these out if we can fit them in the contract size?

We can always still remove in the future, but this would allow us to run the tests without having to manually uncomment every time?

Reply:
If we leave them they need restricted access control.

Reply:
These have been removed. Once the test case PR is approved, I can merge it into this branch before merging this into develop.